values.yaml

```diff
@@ -409,6 +409,9 @@ controllerteam:
   args:
   - "controller"
   - "team"
+  image:
+    repository: jenkinsxio/builder-go
+    tag: 0.0.498
```

Comment:
may as well switch to a more recent image while we're at it? e.g. `0.1.113` 
https://hub.docker.com/r/jenkinsxio/builder-go/tags/

Reply:
this is a pure revert from GH.
I don't really want to mix a revert commit with changes.
will do a follow up for that.